google-play-book-downloader-pdf.py

```
@@ -194,11 +194,18 @@ def normalize_windows_cmd_caret(s: str) -> str:
             key, value = header.split(":", 1)
             headers[key.strip()] = value.strip()
 
+    cookies_unparsed = args.cookies
     cookies = {}
-    if not args.cookies:
+    if "Cookie" in headers:
+        if args.cookies:
+            logging.warning(f"Cookies appear in both --cookie argument and headers, check the cURL request in curl.txt")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)